package.json

```
@@ -64,6 +64,7 @@
     "@sentry/utils": "^7.108.0",
     "@spotlightjs/spotlight": "^1.2.13",
     "@tanstack/react-query": "^4.29.7",
+    "@tanstack/react-query-devtools": "4",
```

would prefer the version thats going to be reflected in the lockfile since we upgrade it that way
```suggestion
    "@tanstack/react-query-devtools": "4.36.1",
```

---

Are you thinking of pinning the version to `4.36.1` as well or just having the full semantic version?

---

`^4.36.1` or `4.36.1` is fine. make sure you yarn install again after since it will change the lockfile too